serving/src/main/java/feast/serving/controller/ServingServiceGRpcController.java

```diff
@@ -96,7 +100,11 @@ public void getOnlineFeatures(
         this.checkProjectAccess(request.getFeaturesList());
       }
       RequestHelper.validateOnlineRequest(request);
+      Span span = tracer.buildSpan("getOnlineFeatures").start();
```

Comment:
Can you explain the motivation/difference in this change? Where would a spike in authorization show up?

Reply:
The interceptor will automatically create a span surrounding the entire GetOnlineFeatureRequest calls. Any span that is created within will be a child span. The end result looks something like:

GetOnlineFeatureRequest  |------------------------------------------------------------------|
getOnlineFeature                                    |-------------------------------------------|

So the time between the start of GetOnlineFeatureRequest and the start of getOnlineFeature will be the time spent on authorization, validation, etc.

We can add spans for authorization as well, in which case it will look like:

GetOnlineFeatureRequest  |------------------------------------------------------------------|
authorization                            |-----|
getOnlineFeature                                    |-------------------------------------------|

So it really depends on how fine grained we want the span to be like. I am fine with adding more span and possibly renaming the operation name, if you think that's better?